packages/components/src/Accordion/Accordion.tsx

```diff
@@ -0,0 +1,151 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *
+ * OpenCRVS is also distributed under the terms of the Civil Registration
+ * & Healthcare Disclaimer located at http://opencrvs.org/license.
+ *
+ * Copyright (C) The OpenCRVS Authors. OpenCRVS and the OpenCRVS
+ * graphic logo are (registered/a) trademark(s) of Plan International.
+ */
+import React from 'react'
+import styled from 'styled-components'
+
+const Container = styled.div`
+  box-sizing: border-box;
+  border-bottom: 1px solid ${({ theme }) => theme.colors.grey200};
+  padding-bottom: 16px;
+  width: 100%;
+
+  details > summary::before {
+    border-style: solid;
+    border-width: 3px 3px 0 0;
+    content: '';
+    display: inline-block;
+    height: 8px;
+    position: relative;
+    top: 7px;
+    vertical-align: top;
+    width: 8px;
+    left: 0;
+    margin-right: 10px;
+    transform: rotate(45deg);
+  }
```

Comment:
Could we use chevron `<Icon />` and maybe hide this browser default one?

Reply:
we don't need the rotate animation also